rts/lra/run.sh

```diff
@@ -59,32 +67,20 @@ function getDebugArgs {
 
 cd "$( dirname "${BASH_SOURCE[0]}" )"
 
-case "$(uname)" in
-   CYGWIN*) export NARAYANA_INSTALL_LOCATION=`cygpath -w $(pwd)/narayana-full-7.0.3.Final-SNAPSHOT` ;;
-   *)       export NARAYANA_INSTALL_LOCATION=$(pwd)/narayana-full-7.0.3.Final-SNAPSHOT ;;
-esac
-
-rm -rf $NARAYANA_INSTALL_LOCATION
-NARAYANA_ZIP="narayana-full-7.0.3.Final-SNAPSHOT-bin.zip"
-[ ! -f "$WORKSPACE/$NARAYANA_ZIP" ] &&\
-   echo "There is no Narayana zip at \$WORKSPACE directory at '$WORKSPACE/$NARAYANA_ZIP" && exit 1
-unzip "$WORKSPACE/$NARAYANA_ZIP"
-
 CURL_IP_OPTS=""
 IP_OPTS="${IPV6_OPTS}" # use setup of IPv6 if it's defined, otherwise go with IPv4
 if [ -z "$IP_OPTS" ]; then
   IP_OPTS="-Djava.net.preferIPv4Stack=true -Djava.net.preferIPv4Addresses"
   CURL_IP_OPTS="-4"
 fi
 
-echo "Narayana installed location = $NARAYANA_INSTALL_LOCATION"
-  java ${IP_OPTS} -Dquarkus.http.port=8080 $(getDebugArgs $PORT) -jar $NARAYANA_INSTALL_LOCATION/rts/lra/lra-coordinator-runner.jar &
+  java ${IP_OPTS} -Dquarkus.http.port=8080 $(getDebugArgs $PORT) -jar $WORKSPACE/rts/lra-examples/coordinator-quarkus/target/lra-coordinator-quarkus-runner.jar &
   ID1=$!
 ((PORT++))
-  java ${IP_OPTS} -Dquarkus.http.port=8081 $(getDebugArgs $PORT) -jar $NARAYANA_INSTALL_LOCATION/rts/lra/lra-coordinator-runner.jar &
+  java ${IP_OPTS} -Dquarkus.http.port=8081 $(getDebugArgs $PORT) -jar $WORKSPACE/rts/lra-examples/coordinator-quarkus/target/lra-coordinator-quarkus-runner.jar &
   ID2=$!
 ((PORT++))
-  java ${IP_OPTS} -Dquarkus.http.port=8082 $(getDebugArgs $PORT) -jar hotel-service/target/quarkus-app/quarkus-run.jar &
+  java ${IP_OPTS} -Dquarkus.http.port=8082 -Dlra.http.port=8080 $(getDebugArgs $PORT) -jar hotel-service/target/quarkus-app/quarkus-run.jar &
```

Comment:
@marcosgopen I don't understand why the hotel-service has a different `lra.http.port` to the flight-service but that could be my lack of familiarity with this area. I won't block on it and will assume it's intentional but mentioning this here.

Reply:
Thanks Tom for this comment, in the demo there are two lra-coordinators and the services communicate to different lra-coordinators (hotel-service and trip-controller to the first and flight-service to the second) .  
I wanted to specify the lra.http.port for the hotel-service too for clarity. It is not important if the port is 8080 or 8081, it is good that both coordinator are in use during the demo.

Reply:
Thank you!